cluster-logging/base/clusterlogging.yaml

```diff
@@ -17,7 +17,10 @@ spec:
     elasticsearch:
       resources:
         requests:
-          cpu: 4
+          cpu: 800m
+          memory: 10Gi
+        limits:
+          cpu: 1200m
```

Comment:
Can we set the cpu limit to 2 instead?
or I think we can remove the limits completely because in case it wants to use more cpu and it's available we want it to use it.

Reply:
Sure, why not.. I don't actually think Elastic is that effective in multicore, so anything above 1 should be rather scaled to multiple replicas.. But why not. :+1: